### PR TITLE
Add Play 2.8 modules

### PR DESCRIPTION
**Pull request by AndyKirsch**
Added new Play 2.8 compatible modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ implicits and tools for:
 
 | play version | scala versions          | scalatest version | artifact name          | bintray                                                                                                                                                                                     |
 |--------------|-------------------------|-------------------|------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 2.8.x        | 2.13.1, 2.12.6, 2.11.12 |                   | play28-json-ops-v4     | [ ![Download](https://api.bintray.com/packages/rallyhealth/maven/play28-json-ops-v4/images/download.svg) ](https://bintray.com/rallyhealth/maven/play28-json-ops-v4/_latestVersion)         |
```
Comment (arkban) [inline README.md:24]: These images don't work, are they supposed to before merging?

File: build.sbt [hunk 2]
```diff
@@ -162,19 +172,20 @@ def playJsonTests(scalacVersion: String, includePlayVersion: String, includeScal
...
   commonProject(id, projectPath, scalacVersion).settings(
     Test / scalacOptions -= "-deprecation",
-    
-    //Suppress semver check since we add a new artifact in v3.2.0
+
+    //Suppress semver check for new play 2.8 modules
     semVerEnforceAfterVersion := {
       (scalacVersion, includePlayVersion, includeScalaCheckVersion) match {
-        case (Scala_2_11, Play_2_5, ScalaCheck_1_14) => Some("3.3.0")
+        case (_, Play_2_8, _) => Some("4.1.0")
```
Comment (arkban) [inline build.sbt:184]: Q: Does it make sense to make a constant for `"4.1.0"`?  I see a lot of constants and literals in here so I'm not sure.

File: build.sbt [hunk 3]
```diff
@@ -206,6 +217,12 @@ def playJsonTests(scalacVersion: String, includePlayVersion: String, includeScal
     case (Scala_2_13, Play_2_7) => Seq(
       `play27-json-ops-213`
     )
+    case (Scala_2_12, Play_2_8) => Seq(
+      `play28-json-ops-212`
+    )
+    case (Scala_2_13, Play_2_8) => Seq(
+      `play28-json-ops-213`
+    )
     case _ => Seq()
```
Comment (arkban) [inline build.sbt:226]: Q: Is this an invalid state? Maybe it should throw or something if it's invalid?

**Review by arkban — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AndyKirsch) [reply README.md:24]: AFAICT they get generated as part of the release process.

File: build.sbt (hunk 3 above)
Comment (AndyKirsch) [reply build.sbt:226]: I think so, but I'm not certain.

File: build.sbt (hunk 2 above)
Comment (AndyKirsch) [reply build.sbt:184]: I don't think so because as soon as this get released it could all get removed

**Review by jeffmay — APPROVED** [review]
Thanks!

**Pull request merged**